python/ql/lib/semmle/python/dataflow/new/internal/TaintTrackingPrivate.qll

```diff
@@ -55,6 +56,8 @@ private module Cached {
     awaitStep(nodeFrom, nodeTo)
     or
     asyncWithStep(nodeFrom, nodeTo)
+    or
+    FlowSummaryImpl::Private::Steps::summaryLocalStep(nodeFrom, nodeTo, false)
```

Comment:
I assume this is also done by other languages that utilize flow-summaries?

Reply:
Yes, and in hindsight, it should have been added with the summary work; I think I just overlooked it back then..